ontopy/excelparser.py

```diff
@@ -282,9 +282,12 @@ def create_ontology_from_pandas(  # pylint:disable=too-many-locals,too-many-bran
                     )
 
     # Add properties in a second loop
+    
```

Comment:
```suggestion

```
Avoid adding blanks to end of lines. If you do `pip install pre-commit` and then `pre-commit install` a set of pre-commit tests and code cleaners will automatically be run before you commit.